### ISSUE TITLE
OCR processing doesn't work

### ISSUE THREAD

**Issue by LauraErhard** [locdb/locdb-frend#325]
We uploaded new files and not one of them gets OCR processed. Is there a problem right now with the OCR component? Can you see anything. Because it is quite strange that even the files with only 2 pages don't get processed. We manually triggered them multiple times already and nothing seems to help.

**Comment by LauraErhard** [comment]
Any news on this?

**Comment by lgalke** [comment]
No idea, in the demo front-end some scans got successfully processed. So it is hard to say why those ones do fail.

Guess the plan is to integrate the new OCR component and make sure everything works with that one.

**Comment by anlausch** [comment]
Just to make clear: All our system versions are still using the old OCR version, as there is an issue with the retrieval of the generated pictures (@rtahseen is working on this). Tahseen will notify us as soon as he has solved the issue. In the meantime we have two options (I proposed these to Annette): a) Using the old component (I decided with Lukas to keep this for now) b) using the new one, but without being able to show the references properly.
